aviary/interface/test/test_height_energy_mission.py

```diff
@@ -101,6 +102,8 @@ def setUp(self):
         self.make_plots = False
         self.max_iter = 100
 
+        _clear_problem_names()  # need to reset these to simulate separate runs
```

Comment:
I vaguely remember this from OpenMDAO. Is this needed to prevent an error when you run a second problem with the same name in a process?

Reply:
yes!